pipelines/migration/projeto_subsidio_sppo/constants.py

```diff
@@ -709,3 +710,289 @@ class constants(Enum):  # pylint: disable=c0103
+                "description": "Todos os dados de realocação foram capturados"
+            }
+        },
+        "sppo_registros": {
+            "check_gps_capture__sppo_registros": {
+                "description": "Todos os dados de GPS foram capturados"
+            }
+        },
+        "gps_sppo": {
+            "check_gps_treatment__gps_sppo": {
+                "description": "Todos os dados de GPS foram devidamente tratados"
+            }
+        },
+        "sppo_veiculo_dia": {
+            "dbt_utils.unique_combination_of_columns__data_id_veiculo__sppo_veiculo_dia": {
+                "description": "Todos os dados de status dos veículos foram devidamente tratados"
+            }
+        },
+    }
+
+    SUBSIDIO_SPPO_POS_CHECKS_LIST = {
+        "sumario_faixa_servico_dia_pagamento": {
+            "not_null__data__sumario_faixa_servico_dia_pagamento": {
+                "description": "Todos os valores da coluna data não nulos"
+            },
+            "not_null__valor_penalidade__sumario_faixa_servico_dia_pagamento": {
+                "description": "Todos os valores da coluna valor_penalidade não nulos"
+            },
+            "not_null__tipo_dia__sumario_faixa_servico_dia_pagamento": {
+                "description": "Todos os valores da coluna tipo_dia não nulos"
+            },
+            "not_null__consorcio__sumario_faixa_servico_dia_pagamento": {
+                "description": "Todos os valores da coluna consorcio não nulos"
+            },
+            "not_null__servico__sumario_faixa_servico_dia_pagamento": {
+                "description": "Todos os valores da coluna servico não nulos"
+            },
+            "not_null__viagens_faixa__sumario_faixa_servico_dia_pagamento": {
+                "description": "Todos os valores da coluna viagens_faixa não nulos"
+            },
+            "not_null__km_apurada_faixa__sumario_faixa_servico_dia_pagamento": {
+                "description": "Todos os valores da coluna km_apurada_faixa não nulos"
+            },
+            "not_null__km_subsidiada_faixa__sumario_faixa_servico_dia_pagamento": {
+                "description": "Todos os valores da coluna km_subsidiada_faixa não nulos"
+            },
+            "not_null__km_planejada_faixa__sumario_faixa_servico_dia_pagamento": {
+                "description": "Todos os valores da coluna km_planejada_faixa não nulos"
+            },
+            "not_null__pof__sumario_faixa_servico_dia_pagamento": {
+                "description": "Todos os valores da coluna pof não nulos"
+            },
+            "not_null__valor_a_pagar__sumario_faixa_servico_dia_pagamento": {
+                "description": "Todos os valores da coluna valor_a_pagar não nulos"
+            },
+            "dbt_utils.accepted_range__viagens_dia__sumario_faixa_servico_dia_pagamento": {
+                "description": "Todos os valores da coluna viagens_dia maiores ou igual a zero"
```

Comment:
```suggestion
                "description": "Todos os valores da coluna `viagens_dia` maiores ou iguais a zero"
```

Alterar para todos os casos semelhantes